docs/whatsnew/1.2.rst

```diff
@@ -29,7 +29,18 @@ smaller improvements and bug fixes, which are described in the
 Solar system ephemerides
 ------------------------
 
-
+Barycentric and apparent positions of some of the major solar-system bodies
```

Comment:
Right now this sounds like a rather obscure feature because it says "barycentric and apparent positions" which *seems* rather technical.  I wonder if instead it should be something like:

Positions of the major solar-system bodies, including the moon, can now be calculated.  These integrate with the coordinates subpackage, allowing use of apparent (e.g. `AltAz`) positions or barycentric values.  These can be calculated using ...